validators/RequestSchema.js

```diff
@@ -11,6 +11,51 @@ const ItemSchema = FileSchema.keys({
   children: Joi.array().items(Joi.string()),
 })
 
+// Contact Us
+const UpdateContactUsSchema = Joi.object({
+  content: Joi.object({
+    frontMatter: Joi.object({
+      layout: Joi.string().required(),
+      title: Joi.string().required(),
+      permalink: Joi.string().required(),
+      feedback: Joi.string(),
```

Comment:
based on what you've said previously, what happens here when an empty feedback section is submitted? would there be a property like so: `{feedback: ""}`, thus causing this to fail validation?

Reply:
Thanks for catching - this would cause a failed validation, I've updated it to allow for empty feedback in https://github.com/isomerpages/isomercms-backend/pull/349/commits/13492c4cc615c662479433ccd1d25b6f342fdc08